torch/_dynamo/variables/lists.py

```diff
@@ -726,17 +726,10 @@ def _listvariable_unflatten(values: List[Any], context: pytree.Context) -> ListV
     return ListVariable(values, mutable_local=MutableLocal())
 
 
-def _register_dynamo_list_to_tree_spec():
-    pytree._register_pytree_node(
-        ListVariable,
-        _listvariable_flatten,
-        _listvariable_unflatten,
-    )
-
-    fx_pytree.register_pytree_flatten_spec(
-        ListVariable,
-        _listvariable_flatten,
-    )
+pytree._register_pytree_node(
+    ListVariable, _listvariable_flatten, _listvariable_unflatten
+)
+fx_pytree.register_pytree_flatten_spec(ListVariable, _listvariable_flatten)
```

Comment:
ListVariable, etc, shouldn't be pytree nodes. We made them pytree nodes in the previous design, but in the new design (where we are tracing through pytrees) these do not need to be pytree nodes anymore